omod/src/main/webapp/pages/login.gsp

```diff
@@ -9,6 +9,9 @@
     <title>${ ui.message("referenceapplication.login.title") }</title>
     <link rel="shortcut icon" type="image/ico" href="/${ ui.contextPath() }/images/openmrs-favicon.ico"/>
     <link rel="icon" type="image/png\" href="/${ ui.contextPath() }/images/openmrs-favicon.png"/>
+    <!-- Latest compiled and minified CSS -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
```

Comment:
Will this work without an internet connection?

Reply:
locally referenced